app/templates/components/event-card.hbs

```diff
@@ -23,10 +23,18 @@
       </SmartOverflow>
       <div class="meta">
         <span class="date">
-          {{general-date this.event.startsAt tz=this.event.timezone}}
+          {{general-date this.event.startsAt 'date-time-long'}}
```

Comment:
also change date-time-long -> date-time-tz-long

Reply:
@iamareebjamal shouldn't we do this also?

Reply:
yes